src/command_modules/azure-cli-interactive/azclishell/configuration.py

```diff
@@ -13,7 +13,7 @@
 
 SELECT_SYMBOL = {
     'outside': '#',
-    'query': '?',
+    'query': '??',
```

Comment:
Yes. It is perfectly valid argparse syntax to use "--foo=bar" instead of "--foo bar". Your parsing needs to be able to handle "--foo=?bar" in addition to "--foo ?bar"